src/js/body.js

```diff
@@ -1 +1,31 @@
-yepnope.injectJs('public/google-analytics.js');
\ No newline at end of file
+yepnope.injectJs('public/google-analytics.js');
+
+//GOOGLE MAP
+var map;
+function initialize() {
+    var mapCanvas = document.getElementById('map-canvas');
+    var mapOptions = {
+        scrollwheel: false,
+        navigationControl: false,
+        mapTypeControl: false,
+        scaleControl: false,
+        center: new google.maps.LatLng(52.248421,21.066009),
+        zoom: 17,
+        mapTypeId: google.maps.MapTypeId.ROADMAP
+    }
+    map = new google.maps.Map(mapCanvas, mapOptions);
+    var marker = new google.maps.Marker({
+        position: new google.maps.LatLng(52.248421, 21.066009),
+        map: map,
+        title:"CODEPOT will be here!"
+    });
+}
+
+google.maps.event.addDomListener(window, 'load', initialize);
```

Comment:
Google Map shouldn't be loaded before user wanted to. At least for mobile version. You can try to guess if user is using mobile device by this script:
```
if (Modernizr.mq('(min-width: ' + 641 + 'px)')) {
// desktop
} else {
// mobile
}
```